### PR TITLE
Feature/issue 43 form profile

### LINKED ISSUE #43: Form
**Priority :**
1

**Duration :** 
0.5

**Description:**
Creation of the form for edit profile

**File added / modified:**

**Additional context**

Add any other context or screenshots here.

### PR DESCRIPTION
**Pull request by vanmarcke**
Issue [#43]

Creation of the form for edit profile

### CODE REVIEW

File: src/Form/ProfileFormType.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\File;
+use Symfony\Component\Validator\Constraints\Length;
+
+class ProfileFormType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+        ->add('newavatar', FileType::class, [
```
Comment (antoinetrouve) [inline src/Form/ProfileFormType.php:21]: indentation ?

File: src/Manager/FigureManager.php [hunk 2]
```diff
@@ -20,6 +20,16 @@ public function __construct(private SnowFigureRepository $snowFigureRepository,
         $this->slugger = $slugger;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getListFigures(): array
+    {
+        $figuresList = $this->snowFigureRepository->findBy([], ['id' => 'DESC']);
```
Comment (antoinetrouve) [inline src/Manager/FigureManager.php:28]: en une seule ligne ? 
```suggestion
        return $this->snowFigureRepository->findBy([], ['id' => 'DESC']);
```

File: src/Manager/FigureManager.php [hunk 3]
```diff
@@ -20,6 +20,16 @@ public function __construct(private SnowFigureRepository $snowFigureRepository,
...
+
+        return $figuresList;
```
Comment (antoinetrouve) [inline src/Manager/FigureManager.php:30]:
```suggestion
        
```

**Review by antoinetrouve — APPROVED** [review]

File: src/Form/ProfileFormType.php (hunk 1 above)
Comment (vanmarcke) [reply src/Form/ProfileFormType.php:21]: done

File: src/Manager/FigureManager.php (hunk 2 above)
Comment (vanmarcke) [reply src/Manager/FigureManager.php:28]: done

File: src/Manager/FigureManager.php (hunk 3 above)
Comment (vanmarcke) [reply src/Manager/FigureManager.php:30]: done

**Pull request merged**